doc/src/sphinx/Names.rst

```diff
@@ -161,6 +161,23 @@ is bound by Finagle to the Internet address ``localhost:8080``. Similarly,
 is the path describing the serverset_ ``/foo/bar`` on the ZooKeeper
 ensemble ``zk.local.twitter.com:2181``.
 
+Dtabs may contain line-oriented comments beginning with ``#``.  For
```

Comment:
maybe add a note that `#` must be preceded by a space or must be the first thing in a line?